build/dependencies.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
 
   # agnhost: bump this one first
   - name: "agnhost"
-    version: "2.28"
+    version: "2.29"
```

Comment:
@liggitt do you remember the steps to make sure this image is built/pushed?

I think we also have to follow up in a subsequent PR to bump the version in `initImageConfigs`:  https://github.com/kubernetes/kubernetes/blob/96be00df69390ed41b8ec22facc43bcbb9c88aae/test/utils/image/manifest.go#L219

Reply:
I am familiar with the process. There will be a postsubmit job triggered after this PR is merged, which builds and pushes the image to a staging registry. Then I will have to promote it and open a subsequent PR to update the file you mentioned.